vimmasite/vimma/celeryconfig.py

```diff
@@ -5,6 +5,10 @@
 _every_5min = crontab(minute='*/5')
 _every_1h = crontab(minute=0)
 
+BROKER_BACKEND = "redis"
```

Comment:
One less daemon to run and install. Redis is more often used than Rabbitmq (atleast in the projects I've done). Rabbitmq can be used for production still.